upgrade-notes/storefront_20241003_113606.md

```diff
@@ -0,0 +1,4 @@
+#### unified date format based on localization ([#3471](https://github.com/shopsys/shopsys/pull/3471))
+
+-   unified date format based on localization
+-   specific localization needs to be imported in file formatDate.ts from
```

Comment:
I would include an example of the import as it is in the stack overflow for better understanding eg. `import 'dayjs/locale/en-gb';`